installer.sh

```diff
@@ -12,15 +12,8 @@ INSTALL_DIR=$(pwd)/.dist
 PACKAGE_DIR=$(pwd)/.packages
 
 MSWEBVIEW_VERSION=1.0.3351.48
-MSWEBVIEW_INCLUDE_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/include
-MSWEBVIEW_LIB_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/x64
-
-
-
 CHROMIUM_VERSION=138.0.17
-CHROMIUM_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include
-CHROMIUM_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64
-
+BROWSER_OPTIONS="<Chromium|MSWebView2>"
```

Comment:
Prefer lower case `<chromium|mswebview2>`

This is MUCH handier and easier for user input options.